pyomo/util/report_scaling.py

```diff
@@ -89,11 +71,10 @@ def _check_coefficents(comp, expr, too_large, too_small, largs_coef_map, small_c
     ders = reverse_sd(expr)
     for _v, _der in ders.items():
         if isinstance(_v, _GeneralVarData):
```

Comment:
When is v not a `_GeneralVarData`?

Reply:
If v is not a `_GeneralVarData`, then it is a `ParamData` or some other node in the expression tree that we do not care about in this context.